CSharp_Course_Project/Task1_Calculation.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharp_Course_Project
+{
+    class Task1_Calculation
+    {
+        public static double Calculation()
+        {
+            while (true)
+            {
+                Console.Write("Enter the first argument: ");
+                string firstArgString = Console.ReadLine();
+                
+                Console.Write("Enter the second argument: ");
+                string secondArgString = Console.ReadLine();
+                
+                Console.WriteLine("Enter one of the option: '+' '-' '*' '/'");
+                bool isValidSymbol = Char.TryParse(Console.ReadLine(), out char action);
+                if (!InputValueCheck.IsNumber(firstArgString) || !InputValueCheck.IsNumber(secondArgString))
+                {
+                    Console.WriteLine("Argument is not a number. Try again...");
+                    continue;
+                }
+                else if (!Task1_Calculation.IsValidSymbol(action))
+                {
+                    Console.WriteLine("Not valid symbol. Try again...");
+                    continue;
+                }
+                double firstArg = Convert.ToDouble(firstArgString);
+                double secondArg = Convert.ToDouble(secondArgString);
+
+                double result = Task1_Calculation.GetMathResult(firstArg, secondArg, action);
+                Console.WriteLine($"Result: {result}");
+                return result;
+            }
+        }
+
+        private static double GetMathResult(double firstArg, double secondArg, char action)
+        {
+            double result = 0;
+            switch (action)
+            {
+                case '+':
+                    result = Add(firstArg, secondArg);
```

Comment:
We should specify class name if invoke a static member.

Reply:
fixed